### DISCUSSION TITLE
[@types/assets-webpack-plugin] depends on @types/webpack 4 not 5

### DISCUSSION THREAD

**Discussion by palfrey** [DefinitelyTyped/DefinitelyTyped#56527] [Issues with a @types package]
@types/webpack 5 came out 7 months ago, but the @types/assets-webpack-plugin still depends on the older one, which causes issues in typescript due to both versions of @types/webpack getting loaded

[a comment by an automated account is not shown]

**Comment by kryops** [comment]
Somehow I thought that https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51712 would have solved this. I didn't realize that while a passthrough for webpack 5 was added, the existing typings were still pinned to webpack 4.

This seems to affect a lot of other typings as well: While I only found 15 definitions targeting webpack 5, there are 114 still targeting webpack 4.

I will have a look over the webpack definitions I own over the next few days and try to update them to webpack 5; especially as most current major versions of the plugins only support webpack 5 it really does not make sense to pin the types to webpack 4 ☺️

**Reply by kryops** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56561
